includes/functions-vstools-create.md

```diff
@@ -1,7 +1,7 @@
 The Azure Functions project template in Visual Studio creates a project that can be published to a function app in Azure. A function app lets you group functions as a logic unit for easier management, deployment, and sharing of resources.   
 
-1. Right mouse click on the project node in **Solution Explorer**, then choose **Add** > **New Item**. Choose **Azure Function** from the dialog box.
+1. Right mouse click on the solution  node in **Solution Explorer**, then choose **Add** > **New Project...**.
```

Comment:
New Function App project is added under the Solution node, not the Project node:
https://i.stack.imgur.com/vjMGE.png